server/src/main/resources/sql_features.tsv

```diff
@@ -176,6 +176,7 @@ F051	Basic date and time	5	Explicit CAST between datetime types and character st
 F051	Basic date and time	6	CURRENT_DATE	NO		current_timestamp
 F051	Basic date and time	7	LOCALTIME	NO		
 F051	Basic date and time	8	LOCALTIMESTAMP	NO		
+F051	Basic date and time	9	DATE data type and CURDATE scalar function	NO		
```

Comment:
Does this sub-feature exists? PostgreSQL only lists 8 subfeatures for `F051`. 
But what about subfeature `1` and `6`?

Reply:
Sorry, I misunderstood the pattern here - was checking other commits and thought it's kind of registry of added functions. Will address that - basically, it's about changing 1 and 6 from NO to YES but before I had to double check that I did another things correctly.

Reply:
Removed line with that extra feature but haven't changed 1 and 6 features to YES - not sure whether there is an extra step besides that PR to count them activated.

Reply:
It's fine (and also preferable) to do that with the commit which is implementing the features (so this PR).

Reply:
Done